kubernetes/client.go

```diff
@@ -149,6 +149,11 @@ type IstioClient struct {
 	// See istio_details_service.go#HasSecurityResource() for more details.
 	securityResources *map[string]bool
 
+	// securityResources private variable will check which resources kiali has access to from security.istio.io group
+	// It is represented as a pointer to include the initialization phase.
+	// See istio_details_service.go#HasSecurityResource() for more details.
```

Comment:
Minor: Update the comments, are copied from securityResources var and this is authenticationResources with also uses a different method.